escher/scene/model.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include "escher/scene/object.h"
+
+#include <memory>
+#include <vector>
+
+namespace escher {
+
+class Model {
```

Comment:
nit: I'd recommend Splitting this into two pieces: ModelBuilder and Model.  That way Model can be immutable.